JustSaying/JustSayingBus.cs

```diff
@@ -200,23 +205,18 @@ private void Publish(IMessagePublisher publisher, Message message, int attemptCo
             }
             catch (Exception ex)
             {
-                if (Monitor == null)
-                    Log.Error("Publish: Monitor was null - duplicates will occur!");
-
-                if (attemptCount == Config.PublishFailureReAttempts)
+                if (attemptCount >= Config.PublishFailureReAttempts)
                 {
                     Monitor.IssuePublishingMessage();
-
-                    var errorMessage = "Unable to publish message " + message.GetType().Name;
-                    Log.Error(ex, errorMessage);
+                    Log.Error(ex, $"Unable to publish message {message.GetType().Name} after {attemptCount} attempts");
                     throw;
                 }
 
-                Thread.Sleep(Config.PublishFailureBackoffMilliseconds * attemptCount); // ToDo: Increase back off each time (linear)
+                Thread.Sleep(Config.PublishFailureBackoffMilliseconds * attemptCount);

```

Comment:
Yeah, no harm it in. It would be nice to be aware of transient connectivity issues, at the moment there's no knowing if it's working every-time or not.